tests/compat-tls13.sh

```diff
@@ -0,0 +1,1356 @@
+#!/bin/bash
```

Comment:
It would better not to duplicate compat.sh or ssl-opt.sh (I have not checked) what this script is the closest to.

Reply:
Removed